Semi Auto Magnum pistol.

ShockOmega opened the pull request:
Added sprites for in hands, base, icon
Added new yml code for the magazine and weapon itself 
Need to -> Add more mag state lights

## About the PR 
Created the sprites and yml code for a new .45 magnum Semi-Automatic handgun. It is a large caliber pistol created with the purpose of being used by the Warden or HoS. It has a 12 round capacity, shoots 5 rounds per second, and fires .45 mag.
**Media**

![image](https://user-images.githubusercontent.com/126544617/221812356-49dc89f5-bcfe-4e2a-bec4-d96d2f687a32.png)
![image](https://user-images.githubusercontent.com/126544617/221812434-2dc9bc7b-f68c-40f8-af36-ef21ea72c43f.png)

ShockOmega committed:
Semi Auto Magnum pistol.
Added sprites for in hands, base, icon
Added new yml code for the magazine and weapon itself
Need to -> Add more mag state lights

ShockOmega committed:
Added new mag state lights

ShockOmega committed:
Added and changed Mk45c gun shots and mag eject and inject sounds.

DEATHB4DEFEAT commented on Resources/Textures/SimpleStation14/Objects/Weapons/Guns/Pistols/mk45c.rsi/meta.json:
```diff
@@ -0,0 +1,76 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "ShockOmega",
```
```suggestion
    "copyright": "DSC@ShockOmega#2505 (297567991112007681)",
```

ShockOmega replied:
I have changed the copyright in the json file. It is now my discord  user+tag and my ID as suggested. Thanks!

DEATHB4DEFEAT reviewed (changes requested):
cool

ShockOmega committed:
-> Removed unneeded sprite for a different gun.

ShockOmega committed:
-> Added magazine to the light layers.
-> When magazine is ejected, the gun sprite now looks like it has been ejected.
-> Gun lights turn off when magazine is ejected.

ShockOmega committed:
-> Fixed typo in copyright.
